src/api/devices/index.js

```diff
@@ -18,3 +18,10 @@ export const getDeviceHasUpdate = async (id) => {
 export const updateDeviceLatestImage = async (payload) => {
   return await instance.post(`${EDGE_API}/updates`, payload);
 };
+
+export const updateSystemByCommitID = async (payload) => {
+  return await instance.post(`${EDGE_API}/updates`, {
+    CommitID: payload.CommitID,
+    DevicesUUID: payload.DevicesUUID,
+  });
+};
```

Comment:
`updateSystemByCommitID` is the same as `updateDeviceLatestImage`. You can just call `updateDeviceLatestImage` with the commit and device ids. (It should also probably be renamed since it can be used to update the device to any version, not just the latest.)